src/pages/workspace/expensifyCard/issueNew/IssueNewCardPage.tsx

```diff
@@ -1,23 +1,29 @@
+import type {StackScreenProps} from '@react-navigation/stack';
 import React, {useEffect} from 'react';
 import {useOnyx} from 'react-native-onyx';
+import type {SettingsNavigatorParamList} from '@navigation/types';
 import type {WithPolicyAndFullscreenLoadingProps} from '@pages/workspace/withPolicyAndFullscreenLoading';
 import withPolicyAndFullscreenLoading from '@pages/workspace/withPolicyAndFullscreenLoading';
 import * as Card from '@userActions/Card';
 import CONST from '@src/CONST';
 import ONYXKEYS from '@src/ONYXKEYS';
+import type SCREENS from '@src/SCREENS';
 import AssigneeStep from './AssigneeStep';
 import CardNameStep from './CardNameStep';
 import CardTypeStep from './CardTypeStep';
 import ConfirmationStep from './ConfirmationStep';
 import LimitStep from './LimitStep';
 import LimitTypeStep from './LimitTypeStep';
 
-function IssueNewCardPage({policy}: WithPolicyAndFullscreenLoadingProps) {
+type IssueNewCardPageProps = WithPolicyAndFullscreenLoadingProps & StackScreenProps<SettingsNavigatorParamList, typeof SCREENS.WORKSPACE.EXPENSIFY_CARD_ISSUE_NEW>;
+
+function IssueNewCardPage({policy, route}: IssueNewCardPageProps) {
     const [issueNewCard] = useOnyx(ONYXKEYS.ISSUE_NEW_EXPENSIFY_CARD);
 
     const {currentStep} = issueNewCard ?? {};
 
     const policyID = policy?.id ?? '-1';
+    const backTo = route.params.backTo;
```

Comment:
Can we access this safely please just in case?
```suggestion
    const backTo = route?.params?.backTo;
```